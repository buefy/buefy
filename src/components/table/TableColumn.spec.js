import { shallowMount } from '@vue/test-utils'
import BTable from '@components/table/Table'
import BTableColumn from '@components/table/TableColumn'

let wrapper

describe('BTableColumn', () => {
    beforeEach(() => {
        wrapper = shallowMount(BTableColumn, {
            parentComponent: BTable
        })
    })

    it('is called', () => {
        expect(wrapper.name()).toBe('BTableColumn')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})
