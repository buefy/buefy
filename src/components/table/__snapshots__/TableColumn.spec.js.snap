// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BTableColumn render correctly 1`] = `<td class=""></td>`;
